.github/ci-test-connections-benchmark.yaml

```diff
@@ -0,0 +1,53 @@
+connections:
```

Comment:
@pankajastro, I'm sure there is a good reason why we're not using the previously existing `ci-test-connections.yml`, but I'd love to know them from you. Could you share them, please?

Reply:
We wanted to have connections with different parameters. But, as discussed offline removed the new configuration file and added a new connection to the existing one.